connectors/MicrosoftGraphSecurity/apidefinition.swagger.json

```diff
@@ -0,0 +1,1386 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Graph Security",
+    "description": "The Microsoft Graph Security connector helps to connect different Microsoft and partner security products and services, using a unified schema, to streamline security operations, and improve threat protection, detection, and response capabilities. Learn more about integrating with the Microsoft Graph Security API at https://aka.ms/graphsecuritydocs",
+    "version": "1.0",
+    "contact": {
+      "name": "team_name",
+      "email": "contact_email"
+    },
+    "x-ms-api-annotation": {
+      "status": "Preview"
+    }
+  },
+  "host": "graph.microsoft.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/v1.0/security/alerts": {
+      "get": {
+        "description": "Get a list of security alerts for this Azure Active Directory tenant. Use with different query parameters.",
+        "summary": "Get alerts",
+        "operationId": "GetAlerts",
+        "x-ms-visibility": "important",
+        "consumes": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml",
+          "application/x-www-form-urlencoded"
+        ],
+        "produces": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "@odata.count": {
+                  "type": "integer",
+                  "format": "int32",
+                  "title": "Alerts count",
+                  "description": "The number of alerts returned"
+                },
+                "value": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/definitions/Alert"
+                  },
+                  "title": "Alerts",
+                  "description": "The alerts returned"
+                },
+                "@odata.nextLink": {
+                  "type": "string",
+                  "title": "Next link",
+                  "description": "A link to get the next results in case there are more results than requested"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request"
+          },
+          "403": {
+            "description": "Unauthorized, get admin access grant"
+          },
+          "500": {
+            "description": "Internal Server Error"
+          },
+          "default": {
```

Comment:
Can we remove this from public version? It is not really needed, also '500' and 'default' may bring a confusion and we should discourage authors to specify them in swagger.